README.md

```diff
@@ -25,10 +25,10 @@ type `quit` to exit the VM. Alternatively, run `systemctl poweroff`.
 To build your own kernel, add the following to `mkosi.conf`:
 
 ```conf
-[Config]
+[Include]
 Include=modules/kernel
 
-[Content]
+[Build]
 BuildSources=<path-to-your-kernel-sources>:kernel
 ```
```

Comment:
I guess this also should be switched around, @DaanDeMeyer? Given that the matches only work if `BuildSources` is set before the include